### DISCUSSION TITLE
Runplan failed with exit code 1

### DISCUSSION THREAD

**Discussion by freppletest** [frePPLe/frepple#405] [Q&A]
Hi,
I've installed frepple 6.16.0 community edition on Debian 18.04 LTS, successfully. Everything seems to work fine, but when I launch "Execute" for a plan, I take: "Failed with exit code 1". So, I've tryied to launch it from the shell with the following result:

![frepple_plan_exitcode_1](https://user-images.githubusercontent.com/91655045/135400674-c7f64976-7fc5-41fb-ad7c-7edabac35e1c.png)

Any suggestion?

Tx in advance

**Comment by jdetaeye** [comment]
> ... on Debian 18.04 LTS.

If you used the ubuntu 18 deb package to install on Debian 18, you are on uncharted domain.
Even if both are ubuntu 18 is based on Debian, there is no guarantee things are compatible.

I assume you compiled from the source code.  I see python 3.9 in your python path.  The default Python version on ubuntu 18 is 3.6, I believe.  
A possible explanation is that you have both Python 3.6 and 3.9 on your machine.  The command line is using 3.9 where you have the django package installed.  The planning engine executable on the other hand is using 3.6, where django isn't installed.   Also installing all frepple dependencies on the python 3.6 version would solve this issue then.

**Comment by freppletest** [comment]
I'll try to adjust dependencies or If I fails, I'll try with Ubuntu.

Thank you very much!

**Reply by jdetaeye** [reply]
Or use docker...
